### DISCUSSION TITLE
[Feature request] Add notification center

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#348] [Feature Requests]
add a placeholder notification center with placeholder switches for settings in the windows 10 style.
the settings can include

- light/dark mode (stuck on dark)
- wifi (shows ethernet connected)
- bluetooth (disabled)
- VPN (disabled)
- battery saver (disabled)
- tablet mode (disabled)
- night light(tints screen a  reddish hue)
- connect(could possibly use presentation API)
- project(shows "PC screen only" and "Duplicate")
- screen snip(if possible to implement a screenshot feature)

**Comment by DustinBrett** [comment]
Thanks for the idea. I am in general against placeholders and I prefer to only add things once they actually function. For that reason things like VPN and WiFi would probably not work. I can read the battery level though and am planning to add that along with a system tray in the near future. I may also be able to read bluetooth devices and tablet mode. As for light/dark, I indeed plan to make a light theme and will have a toggle at that time.
